README.md

```diff
@@ -48,12 +46,3 @@ Build Instructions for Master
 
 5. Open `Blockzilla.xcodeproj` in Xcode.
 6. Build the `Focus` scheme in Xcode.
-
-V8.0 Development Branch
-----------------
-
-For version 8.0 of Focus, we are working off of the v8.0-dev branch.
-
-This branch is written in Swift 4.2. Pull requests for this branch must also be written in Swift 4.2. 
-
-For bugs and features for the upcoming v8.0 release, please see the V8.0 milestone within the GitHub Issues.
```

Comment:
Consider keeping this in elsewhere.

Reply:
Added this information in a different wording.